src/store/policies/actions.ts

```diff
@@ -23,9 +23,12 @@ export const getPoliciesPage =
   };
 
 export const removePolicy =
-  (type: string, guid: string) => async (dispatch: Dispatch) => {
+  (type: string, guid: string, policies) => async (dispatch: Dispatch) => {
+    policies = {
+      body: policies.body.filter((item) => item.guid !== guid),
+      totalCount: policies.totalCount - 1,
+    };
```

Comment:
I did it this way, because after reloading data from API user would be back on 1st page and I think it's better to stay at the same page with same **items per page** value (we don't have current page & items per page information outside table component)